reports/final/3-Technical/2-Details/11-LearningAlgorithm.md

```diff
@@ -47,12 +47,79 @@ session and associating each one with the event that immediately preceded the re
 
 #### Strategy Version 3 {-}
 
-The third and current strategy also looks at all user requests in a training session. Additionally,
+### Strategy
+
+The third strategy also looks at all user requests in a training session. Additionally,
 this strategy uses the state of all sensors at the beginning of the training session to filter out 
 events that did not actually cause a change in the state of the home environment. This strategy 
 triggers a user action based on the first event that indicated a change in the state of the 
 system.
 
+##### Outcome {-}
+
+Using this strategy, the system is able to learn the correct action to take in a given scenario.
+However, it was found that the system often associates user actions with small variations in 
+sensor values, such as temperatures. The effect is that the system often takes action at
+inappropriate times, due to the misidentification of small fluctuations as trigger events.
+
+#### Strategy Version 4 {-}
+
+The fourth and final strategy builds on version three by using data from multiple training sessions
+in order to ignore noise in the training data. This strategy can make use of *negative* training
+sessions. A negative training session is a session in which the user does not perform the 
+action that they are attempting to train the system. Negative training sessions allow this strategy

```

Comment:
maybe say 'A negative training session is a session in which the user does not perform the 
 +action that they are attempting to train the system to perform' or something similar